src/pools/composableStable/composableStablePool.ts

```diff
@@ -50,129 +46,28 @@ export class ComposableStablePool extends PhantomStablePool {
         );
     }
 
-    _exactTokenInForTokenOut(
```

Comment:
Was that simply being duplicated from PhantomStablePool? Is that why we were able to simply remove them?